PowerUp/src/Commands/AutonomousDrive.cpp

```diff
@@ -11,8 +11,8 @@ AutonomousDrive::AutonomousDrive(const RobotNavigation::RobotTrajectory& path) :
 void AutonomousDrive::Initialize()
 {
 	Robot::driveTrain->SetSafetyEnabled(false);
-	Robot::driveTrain->InitializeMotionProfile(m_path.m_left.m_generator, m_path.m_right.m_generator);
 	Robot::driveTrain->EnableVoltageCompensation(true);
+	Robot::driveTrain->InitializeMotionProfile(m_path.m_left.m_generator, m_path.m_right.m_generator);
```

Comment:
I'm guessing this is reordered so the voltage compensation will work correctly?

Reply:
I don't believe it works yet, but it made sense to enable the voltage compensation before starting the profile. I think there may be more to what is wrong - I will find out Wed